frontend/src/views/admin/matching/position-list.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { useThunkDispatch } from "../../../libs/thunk-dispatch";
+import { round } from "../../../libs/utils";
+
+import { PositionSummary } from "./types";
+import { setSelectedPosition } from "./actions";
+
+import { Form } from "react-bootstrap";
+import "./styles.css";
+
+/**
+ * A searchable list of position codes.
+ */
+export function PositionList({
+    selectedPositionId,
+    positionSummaries,
+}: {
+    selectedPositionId: number | null;
+    positionSummaries: Record<number, PositionSummary>;
```

Comment:
`positionSummaries` should have type `PositionSummary[]`. Otherwise it should have a different name.